plugins/aws/resoto_plugin_aws/resource/eks.py

```diff
@@ -22,7 +22,7 @@ def update_resource_tag(self, client: AwsClient, key: str, value: str) -> bool:
                     action="tag_resource",
                     result_name=None,
                     resourceArn=self.arn,
-                    tags=[{key: value}],
+                    tags={key: value},
```

Comment:
These files don't look like they should be part of this PR?

Reply:
Yes - a bug I found on the way. Will extract it into a separate PR.